test/lpdf.cc

```diff
@@ -1,13 +1,13 @@
 #include <gtest/gtest.h>
 
-#include <Eigen/Dense>
 #include <stan/math/prim/fun.hpp>  // lgamma, lmgamma
 #include <stan/math/prim/prob.hpp>
+#include <stan/math/rev.hpp>
 
 #include "algorithm_state.pb.h"
 #include "src/hierarchies/lin_reg_uni_hierarchy.h"
 #include "src/hierarchies/nnig_hierarchy.h"
-#include "src/hierarchies/nnw_hierarchy.h"
+// #include "src/hierarchies/nnw_hierarchy.h"
```

Comment:
Comments to remove

Reply:
Not sure, maybe code to remove?

Reply:
nnw_hierarchy here is not included due to the well known issue while checking logpost  = logprior + loglikelihood - logmarg, I guess.